spec/models/tax_return_spec.rb

```diff
@@ -92,29 +93,24 @@
       expect(result).to have_key("intake")
       expect(result).to have_key("prep")
       expect(result).to have_key("review")
-      expect(result).to have_key("finalize")
-      expect(result).to have_key("filed")
+      expect(result).to have_key("file")
     end
 
     it "includes all intake statuses except before consent" do
-      expect(result["intake"].length).to eq 6
+      expect(result["intake"].length).to eq 5
       expect(result["intake"]).not_to include "intake_before_consent"
     end
 
     it "includes all prep statuses" do
-      expect(result["prep"].length).to eq 4
+      expect(result["prep"].length).to eq 3
```

Comment:
These tests are sort of hard to read. We have some repetition (all prep statuses, all review statuses, etc.). I wonder if we can merge them into one test and/or remove some of them. Not a blocker at all.